_posts/institute-materials/day1_monday/2016-06-20-openNEONH5.md

```diff
@@ -1,26 +1,33 @@
 ---
```

Comment:
No problem. I'll check it out when I run through the PR!
Thanks, Megan
Leah
--
Leah A. Wasser, Ph.D.
Supervising Scientist
Education / Data Products
National Ecological Observatory Network (NEON)
Boulder, Colorado
--------------------
For free tutorials and resources on working with spatio-temporal data:
http://www.neondataskills.org<http://www.neondataskills.org/>

From: "Megan A. Jones" <notifications@github.com<mailto:notifications@github.com>>
Reply-To: NEON-WorkWithData/neon-data-institute-2016 <reply@reply.github.com<mailto:reply@reply.github.com>>
Date: Thursday, June 9, 2016 at 10:54 AM
To: NEON-WorkWithData/neon-data-institute-2016 <neon-data-institute-2016@noreply.github.com<mailto:neon-data-institute-2016@noreply.github.com>>
Subject: Re: [NEON-WorkWithData/neon-data-institute-2016] updated Rmd ONLY for Monday materials (#69)


In _posts/institute-materials/day1_monday/2016-06-20-openNEONH5.md<https://github.com/NEON-WorkWithData/neon-data-institute-2016/pull/69#discussion_r66479008>:

> @@ -1,26 +1,33 @@
>  ---


I'm really confused as to why this is showing up as changed. I haven't modified the YAML at all.

-
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub<https://github.com/NEON-WorkWithData/neon-data-institute-2016/pull/69/files/b8493960bf3e813bf8771c4bb6256bb06c3e3494#r66479008>, or mute the thread<https://github.com/notifications/unsubscribe/AHS3qqDBKufv0h29jNyV7kBBoMyay3eFks5qKEVFgaJpZM4IxerR>.